lesson15/main.js

```diff
@@ -4,8 +4,10 @@ const closeButton = document.getElementById("js-close-button");
 const overlay = document.getElementById("js-overlay");
 const openListButton = document.getElementById("js-open-list-button")
 const ul = document.getElementById("js-ul");
-const numberInput =document.getElementById("js-number-input")
-const validationMessage = document.getElementById("js-validation");
+const form = document.getElementById("js-form");
+const errorMessages = document.getElementsByClassName("js-error-message");
```

Comment:
Q
Why do we get so much in class?
I feel it would be better to unify them since the others were obtained by id.
Just curious if there is any reason for this!

Reply:
this tutorial [freecodecamp tutorial](https://www.freecodecamp.org/news/learn-javascript-form-validation-by-making-a-form/amp/) suggests to use class to get error message <div>s, and I simply followed it after many trials and errors. 
I originally used id to get and add error messages, however it did not work out, so I thought the reason would be using id as it has to be unique and cannot be used in multiple places. 
I can also use the variable in resetInput(), so I thought it might be better, I am not sure what a better practice would be in this case. 

Confirmed the validation function still works when I replaced error message field getting by id. I updated it getting error message field by id for the validation function as Mai-san also suggested a similar point too. 
Please see the commit 
6ad544c96b5c4e21ce5ff7a81fa84174f72e7142